sass/layout/_sidebar-left.scss

```diff
@@ -789,6 +789,10 @@
         padding: 0;
     }
 
+    .SidebarChannelGroup .SidebarChannel.newChannelSpacer {
```

Comment:
Is all of this necessary, or could we get away with simplifying the selector to just `.newChannelSpacer`?

Reply:
we need the nesting to override the existing style.
<img width="443" alt="Untitled" src="https://user-images.githubusercontent.com/4973621/87541018-35632f80-c6be-11ea-9e3d-550ee15ea9d7.png">


we need to override styles to take into account the `height: 0px` property

Reply:
Ugh, I see. Specificity sucks to deal with.